aws/resource_aws_neptune_cluster.go

```diff
@@ -232,6 +236,10 @@ func resourceAwsNeptuneCluster() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Set:      schema.HashString,
 			},
+			"delete_protection": {
```

Comment:
Since the API names this deletion protection, we should follow suit here and everywhere below. 👍 This new attribute also needs documentation in `website/docs/r/neptune_cluster.html.markdown`

```suggestion
			"deletion_protection": {
```